### ISSUE TITLE
Extension doesn't activate after I deleted a profile

### ISSUE THREAD

**Issue by Alexandru-Dumitru** [eclipse/che-che4z-explorer-for-endevor#57]
Extension doesn't activate after I delete a profile (manuall from file system) but the details of that profile are still in settings.json

Steps to replicate:
1. Open extension and add a couple of profiles
2. Make sure they are in settings.json
3. delete a profile yaml file from .zowe/profiles/endevor
4. Reload VS Code 
5. Receive error message: 
```
Activating extension 'broadcomMFD.explorer-for-endevor' failed: Could not find profile named: [PROFILE_NAME]..
```

**Comment by jellypuno** [comment]
The problem is because loadRepositories() is called during initialization. Inside this function it reads the settings.json and then calls the addSession() function if the profile is not added to the tree. I suggest 2 ways to handle this:

1. remove loadRepositories in the the activation and put it somewhere else
2. add a validation that if the profile found in setting.json and also found in allProfiles() then continue with addSession else ignore.

**Comment by Alexandru-Dumitru** [comment]
Thanks for looking into this, Jelly. I would go with the second approach as it seems straightforward, as opposed to discussing the "where" we would move `loadRepositories()` and other associated implications. 

Things to take into consideration (possible enhancements to the process):
- should we add a warning so that the user is aware?
- should we offer to auto fix this ? (can be even an options in endevor settings)
- log these findings (miss-match between settings and actual profiles) in the output channel at least on trace level to help with debugging (really nice to have from the start)

**Comment by jellypuno** [comment]
This is already implemented in v0.12.1
